Tools/Scripts/webkitpy/xcode/simulated_device.py

```diff
@@ -57,15 +59,26 @@ def __init__(self, device_type, use_booted_simulator=True, use_existing_simulato
 
 class SimulatedDeviceManager(object):
     class Runtime(object):
-        def __init__(self, runtime_dict):
-            self.root = runtime_dict['runtimeRoot']
-            self.build_version = runtime_dict['buildversion']
-            self.os_variant = runtime_dict['name'].split(' ')[0]
-            self.version = Version.from_string(runtime_dict['version'])
-            self.identifier = runtime_dict['identifier']
-            self.name = runtime_dict['name']
-
-    AVAILABLE_RUNTIMES = []
+        def __init__(self, runtime_dict: dict):
+            self.root: str = runtime_dict['runtimeRoot']
+            self.bundle_path: str = runtime_dict['bundlePath']
+            self.build_version: str = runtime_dict['buildversion']
+            self.os_variant: str = runtime_dict['platform']
+            self.version: str = Version.from_string(runtime_dict['version'])
+            self.identifier: str = runtime_dict['identifier']
+            self.name: str = runtime_dict['name']
+            self.rebuilt_cache: bool = False
```

Comment:
Nit: Should probably be `has_rebuilt_cache`

Reply:
Done